igvm/hypervisor_preferences.py

```diff
@@ -130,6 +130,36 @@ def __call__(self, vm, hv):
         return value is not None and value > self.limit
 
 
+class HypervisorCpuUsageLimit(object):
+    """Check for CPU usage of the hypervisor incl. the predicted CPU usage
+    of the VM to be migrated.
+
+    Dismiss any hypervisor, which would be above its threshold.
```

Comment:
We cannot really dismiss the hypervisor but just make it least likely to be choosen.

Reply:
Changed the docstring.